packages/compartment-mapper/src/node-modules.js

```diff
@@ -469,6 +492,17 @@ const graphPackages = async (
       `Cannot find package.json for application at ${packageLocation}`,
     );
   }
+
+  // Resolve common dependencies.
+  const commonDependencyDescriptors = {};
+  for (const [alias, dependencyName] of Object.entries(commonDependencies)) {
+    const spec = packageDescriptor.dependencies[dependencyName];
+    commonDependencyDescriptors[dependencyName] = {
+      spec,
```

Comment:
Would prefer to spell out `specifier` in keeping with the prevailing æsthetic.

Reply:
this is not the specifier but the install version spec[ification] eg: `^1.0.0`